readme.md

```diff
@@ -60,9 +60,11 @@ Some good apps written with Electron.
 ## Tools
 
 - [electron-packager](https://github.com/maxogden/electron-packager) - Package and distribute your Electron app for OS X, Linux and Windows (.app, .exe, etc).
+- [electron-builder](https://github.com/loopline-systems/electron-builder) - Build installers.
```

Comment:
I think `Create installers.` would be better as the current sound more like it's installers for builds.